app/components/field.tsx

```diff
@@ -44,10 +44,10 @@ export function Field() {
   }, [size]);
 
   return !plots || sessionState === "dead" ? (
-    <div className="absolute bg-white h-[100dvh] w-[100dvw] z-10">
+    <div className="absolute h-[100dvh] w-[100dvw] z-10 bg-white dark:bg-slate-950">
```

Comment:
I don't think we should put the background color here, since it's already on the `body`... (but feel free to keep the `bg-white` removed here)

```suggestion
    <div className="absolute h-[100dvh] w-[100dvw] z-10">
```

Reply:
this is applying styles to the loader so it needs the background or else the header will show